docs/launch_modes.md

```diff
@@ -70,3 +86,12 @@ Triton Inference Server.
    Analyzer does not currently support profiling remote GPUs. Triton Server in this
    mode needs to be launched with `--model-control-mode explicit` flag to support
    loading/unloading of the models.
+
+### Tradeoffs
+
+| Mode | Pro | Con |
+| - | - | - |
+| *docker* | Don't need Triton Server or its dependencies installed locally | Need Docker installed |
+| *local* | Don't need Docker installed | Need Triton Server and all of its dependencies installed locally |
+| *c_api* | Same as local, but uses faster C API | Need Triton Server and all of its dependencies installed locally |
+| *remote* | Can use Model Analzer on already-running Triton Server | Need Triton Server running on same machine as Model Analyzer |
```

Comment:
Remote mode con is that you cannot use all the Model Analyzer features such as changing model config parameters. All the modes require both the Triton server and MA to be running on the same machine.

Reply:
updated wording 👍